targets/targets.json

```diff
@@ -1735,10 +1742,18 @@
             "MCUXpresso_MCUS",
             "KSDK2_MCUS",
             "FRDM",
-            "Freescale_EMAC"
+            "Freescale_EMAC",
+            "PSA"
         ],
         "is_disk_virtual": true,
-        "macros": ["CPU_MK66FN2M0VMD18", "FSL_RTOS_MBED"],
+        "macros": [
+            "CPU_MK66FN2M0VMD18", 
+            "FSL_RTOS_MBED", 
+            "MBEDTLS_PSA_CRYPTO_C",
+            "MBEDTLS_ENTROPY_NV_SEED",
```

Comment:
This device has TRNG enabling nvseed is redundant

Reply:
Pelion require injection API regardless TRNG